### PR TITLE
Updated the Toast appearance and positioning

### PR DESCRIPTION
**Pull request by knutsynstad**
**Changes:**
- Changes appearance of toast
- Changed position of toast
- Shortened toast message
- Minor variable cleanup

**Before:**

Screenshot 2023-11-02 at 2 34 13 PM
Screenshot 2023-11-02 at 2 34 06 PM

**After:**

Screenshot 2023-11-02 at 2 33 18 PM
Screenshot 2023-11-02 at 2 33 27 PM

### CODE REVIEW

File: src/ui/components/play-icon.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import {unsafeHTML} from 'lit/directives/unsafe-html.js'
 import addOutline from '../assets/icons/add-outline.svg'
 import caretDownOutline from '../assets/icons/caret-down-outline.svg'
 import caretUpOutline from '../assets/icons/caret-up-outline.svg'
+import checkmarkFill from '../assets/icons/checkmark-fill.svg'
```
Comment (niedzielski) [inline src/ui/components/play-icon.ts:9]: This is a nice touch :+1:

File: src/ui/components/play-pen-header.ts [hunk 2]
```diff
@@ -99,7 +99,7 @@ export class PlayPenHeader extends LitElement {
             appearance="orangered"
             size="small"
             icon="share-new-outline"
-            title="Copy Program URL to Clipboard"
+            title="Copy URL to Clipboard"
```
Comment (niedzielski) [inline src/ui/components/play-pen-header.ts:102]: :no_bell: This makes sense to me. Just some background: the original intent here was to indicate that the program / pen is fully represented in the URL which users may not realize. Product banned pen terminology and calling everything a playground is ambiguous.

File: src/ui/components/play-toast.ts [hunk 3]
```diff
@@ -48,10 +65,11 @@ export class PlayToast extends LitElement {
     this.#timeout = globalThis.setTimeout(() => {
       this.opened = false
       this.#timeout = undefined
-    }, 3000)
+    }, 2000)
   }
 
   protected override render() {
-    return html`<slot></slot>`
+    return html`<play-icon size="24px" icon="checkmark-fill"></play-icon
```
Comment (niedzielski) [inline src/ui/components/play-toast.ts:72]: :no_bell: I think this icon should be outside the toast so we can use it for things besides confirmation but not a big deal.

File: src/ui/components/play-toast.ts [hunk 4]
```diff
@@ -12,30 +13,46 @@ declare global {
...
+      user-select: none;
+
+      /* These are the initial opacity and margin values before we animate */
+      opacity: 0;
+      margin-top: var(--space);
+      transition-property: margin opacity;
+      transition-duration: 0.1s;
+      transition-timing-function: ease-out;
+
+      /* RPL/Body Regular/14-BodyReg */
```
Comment (niedzielski) [inline src/ui/components/play-toast.ts:43]: :no_bell: Suggest moving all this stuff into a CSS variable or inheriting.

**Review by niedzielski — APPROVED** [review]

File: src/ui/components/play-pen-header.ts (hunk 2 above)
Comment (knutsynstad) [reply src/ui/components/play-pen-header.ts:102]: That makes sense. I mainly just want it as short as possible. But also not sure if we've used the term "program" elsewhere in the UI.

As for the overall naming, my mental model is:
Play = Product name
Playground = Product category

File: src/ui/components/play-toast.ts (hunk 3 above)
Comment (knutsynstad) [reply src/ui/components/play-toast.ts:72]: Agreed. Making the icon a fixed part of the toast makes the component overall less versatile.

I don't think we'll use toasts for error messages, given it's short display duration, but it's certainly possible for messages of more neutral intent that doesn't warrant the icon.

File: src/ui/components/play-pen-header.ts (hunk 2 above)
Comment (niedzielski) [reply src/ui/components/play-pen-header.ts:102]: My thinking is:

- Play: product
- Playground: execution environment or sandbox; there may be multiple of these on a page like the docs and they each have their own independent state.
- Pen: program and the thing you share.

**Pull request merged**